libs/pages/environments/src/lib/feature/page-general-feature/page-general-feature.tsx

```diff
@@ -18,8 +19,15 @@ export function PageGeneralFeature() {
     selectClustersEntitiesByOrganizationId(state, organizationId)
   )
 
-  const { isLoading, data: environments = [] } = useFetchEnvironments(projectId)
-  const environmentsStatus = useFetchEnvironmentsStatus(projectId)
+  const res = useFetchEnvironments(projectId)
+  let { data: environments = [] } = res
+  const { isLoading } = res
```

Comment:
why didn't you do it `let { data: environments = [], isLoading } = useFetchEnvironments(projectId)` ?

Reply:
Because we want `environments` as `let` but the linter `prefer-const` rule wants `isLoading` as `const`.